setupBenchmark.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -exo pipefail
+
+# Contains OS specific sed function
+. benchmarks/setup/utils.sh
+
+if [ -z $1 ]
+then
+  echo "Please provide a benchmark name! Format should be 'YOUR_NAME-TOPIC'"
+  exit 1
+fi
+benchmark=$1
+
+mvn clean install -DskipTests -T1C
+
+docker build --build-arg DISTBALL=dist/target/zeebe-distribution-*.tar.gz --build-arg APP_ENV=dev -t "gcr.io/zeebe-io/zeebe:$benchmark" .
+docker push "gcr.io/zeebe-io/zeebe:$benchmark"
+
+cd benchmarks/setup/
+
+./newBenchmark.sh "$benchmark"
+
+cd "$benchmark"
+
+# calls OS specific sed inplace function
+sed_inplace 's/camunda\/zeebe/gcr.io\/zeebe-io\/zeebe/' zeebe-values.yaml
+sed_inplace "s/SNAPSHOT/$benchmark/" zeebe-values.yaml
+
+make zeebe starter worker
```

Comment:
from the name of this script it is not entirely clear that this also starts the benchmark. Perhaps it can be something like `setupAndStartBenchmark.sh`

Reply:
I also think this script should be placed in `benchmarks`

Reply:
I can rename it. I put it on purpose on root so I don't need to change the folders etc. It is also more visible and it is easy to use. For the compiling and building the image it is also easier.

Reply:
That makes sense. Just leave it in the root. Perhaps we can still find a better name.

Reply:
@korthout I had hoped setup would it clear enough :D but we can also write `createBenchmark` or `buildAndRunBenchmark` ? What do you think?

Reply:
@Zelldon I think I like createBenchmark the most. I feel like it implies that the benchmark is not just setup but also started.